apps/emqx_connector_aggregator/src/emqx_connector_aggreg_delivery.erl

```diff
@@ -25,14 +25,16 @@
 ]).
 
 -record(delivery, {
-    name :: _Name,
+    id :: aggreg_id(),
     callback_module :: module(),
     container :: emqx_connector_aggreg_csv:container(),
     reader :: emqx_connector_aggreg_buffer:reader(),
     transfer :: transfer_state(),
     empty :: boolean()
 }).
 
+-type aggreg_id() :: {_Type, _Name}.
```

Comment:
Also I feel that on that level it's not really correct to narrow this type that much, it's currently leaking from `emqx_bridge_s3_connector`, which is not part of this library. On this level it's the same as `emqx_connector_aggregator`'s name, which has basically no requirements on how it should look like.